### PR TITLE
docs: Dotnet api quickstart astro

### PR DESCRIPTION
**Pull request by sixhobbits**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205273462533600
  - https://app.asana.com/0/0/1205631932539797
  - https://app.asana.com/0/0/1205631932539799

### CODE REVIEW

File: astro/src/content/quickstarts/quickstart-dotnet-api.mdx [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+Install the .NET JWTBearer package and other supporting libraries. 
+
+```shell
+dotnet add package Microsoft.AspNetCore.Authentication.JwtBearer
+dotnet add package Microsoft.AspNetCore.OpenApi
+dotnet add package System.IdentityModel.Tokens.Jwt
+dotnet add package Microsoft.IdentityModel.Protocols.OpenIdConnect
+```
+
+Next, update your `appsettings.json` file with settings specific to the FusionAuth Issuer and Client Id (Audience).
```
Comment (mark-robustelli) [inline astro/src/content/quickstarts/quickstart-dotnet-api.mdx:121]:
```suggestion
Next, replace your `appsettings.json` file with settings specific to the FusionAuth Issuer and Client Id (Audience).
```

File: astro/src/content/quickstarts/quickstart-dotnet-api.mdx [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+Update the `Program.cs` file to configure ASP.NET authentication to use JWT tokens. Replace the code in the `Program.cs` file with the following code.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-dotnet-api/main/complete-application/Program.cs"
+lang="c#" />
+
+The `Program.cs` file configures the authentication services to use JWT bearer tokens to authenticate users through the `.AddJwtBearer()` method. The Issuer and Audience settings used by the JWT verification code are set in the `appsettings.json` file under the `Authentication` section. Note the `options.Events` section that configures the JWT bearer authentication to use the `OnMessageReceived` event to read the JWT from the `app.at` cookie if it is present. By default, the JWT bearer authentication will only read the JWT from the `Authorization` header. With this code, the JWT can be passed in either the `Authorization` header or the `app.at` cookie.
```
Comment (mark-robustelli) [inline astro/src/content/quickstarts/quickstart-dotnet-api.mdx:136]: The "Note the.." should either be in an Aside or just remove "Note" I.E. The `options.Events` section...

File: astro/src/content/quickstarts/quickstart-dotnet-api.mdx [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+
+This command will build and run the project, which should be running on port 5001.
+
+### Get A Token
+
+There are [several ways to acquire a token](/docs/v1/tech/oauth/) in FusionAuth but for this example, you will use the [Login API](/docs/v1/tech/apis/login) to keep things simple.
+
+First, try the requests as the `teller@example.com` user. Based on the kickstart configuration this user has the `teller` role and should be able to call both `/make-change` and `/panic`.
+
+Acquire an access token for `teller@example` by making the following request.
```
Comment (mark-robustelli) [inline astro/src/content/quickstarts/quickstart-dotnet-api.mdx:245]:
```suggestion
Acquire an access token for `teller@example` by making the following request in a new terminal window.
```

**Review by mark-robustelli — CHANGES_REQUESTED** [review]

File: astro/src/content/quickstarts/quickstart-dotnet-api.mdx (hunk 2 above)
Comment (rideam) [reply astro/src/content/quickstarts/quickstart-dotnet-api.mdx]: removed "Note"

**Review by mark-robustelli — APPROVED** [review]

**Comment by mooreds** [comment]
Closing under https://github.com/FusionAuth/fusionauth-site/pull/2681

**Pull request closed without merging**